test/remote/gateways/remote_hps_test.rb

```diff
@@ -167,6 +167,16 @@ def test_failed_refund
     assert_failure response
   end
 
+  def test_successful_credit
```

Comment:
Similar observation as for the unit tests; would adding in `assert_equal` or `assert_match` lines benefit these tests?

Reply:
The response from HPS doesn't have any valuable information besides the response code and message, which end up in the Response object. That said, `assert_success` only checks that what the `Response.code` is and whether or not it's successful. It's conceivable that the `message` could differ from the `code` we get, so I added an `assert_equal` to verify the the `message` is what we expect.